### DISCUSSION TITLE
Raylib glitching under WSL2

### DISCUSSION THREAD

**Discussion by Dertasz** [raysan5/raylib#3629] [General]
Hello all,

I tried installing and playing with raylib but here is my issue: 
![raylib](https://github.com/raysan5/raylib/assets/19858798/22c99845-af48-4b94-800a-51553dbac631)

The windows display is all weird and flickers. Does anybody knows where it comes from and has a workaround? I am working with WSL2, i dont know if that matters.

**Comment by ghost** [comment]
@Dertasz Check @jcsmurph comment at https://github.com/raysan5/raylib/issues/3630#issuecomment-1852903887. Probably a similar issue.

**Reply by jcsmurph** [reply]
Most likely is.

@Dertasz  Some people have been having luck with changing some config settings for WSL. Check out the link I provided in my issue and see if any of those help.

Unfortunately, they haven't for me so I am going to be using Windows for now.

**Comment by Saurav727** [comment]
I tried playing around with raylib using WSL2 and VSCODE. The window appears but its always black and nothing appears at all.
![image](https://github.com/raysan5/raylib/assets/66196587/0b0e6d22-a2f4-476f-8a36-59999724fb6d)
